gatling/user-files/utils/create-users.js

```diff
@@ -0,0 +1,47 @@
+const
+  rp = require('request-promise'),
+  host = process.argv[2] || 'localhost';
+
+const doit = (id) => {
+  for (let it = 1; it < 100; ++it, ++id) {  
```

Comment:
Modifying a provided parameter is considered bad practice as it reduces maintenability (developers tend to not expect a function argument to be modified).

This is enforced in the Kuzzle main repository by this sonarqube rule: http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1226

Instead, it's more readable to write something like this:

```js
const doit = startingId => {
  for (let it = 1, id = startingId; it < 100; ++it, ++id) {
```

or, better yet, since `it` is not really used:

```js
const doit = startingId => {
  for (let id = startingId; id < startingId + 100; ++id) {
```